runner-manager/cf-driver/worker-setup/README.md

```diff
@@ -0,0 +1,32 @@
+# Worker Setup
+
+The worker setup requires a number of standalone binaries, some packaged in
```

Comment:
praise: Yes, vendoring this for now makes sense. We can build pipelines and something more updatable after.